### ISSUE TITLE
CCDA Service configuration failing

### ISSUE THREAD

**Issue by juggernautsei** [openemr/openemr#7516]
### Describe the bug

write concise description of what the bug is.
Follow these instructions:
https://github.com/openemr/openemr/blob/master/ccdaservice/README.md

### To Reproduce

This happens after running 
![image](https://github.com/openemr/openemr/assets/834958/e9afa214-fcf7-424c-8fee-1a354d04a27c)

1. Add a '....'

### Expected behavior
Install nodejs

### Client configuration

- Browser: Chrome
- OpenEMR version: 7.0.2 (1) 1_addendum
- Operating system: Ubuntu 22.04

**Comment by sjpadgett** [comment]
Not a bug! Sherwin, read the error carefully. The pre built binary is not available for node version 20.15. Clue one!
Plus, I have it set up that if the binary is not downloaded then I request to build the library, however; as the error message points out, build make is not available. Another clue!
So build tools aren't installed or accessible.  But this works for me on windows at least using node 20.15.0.
I haven't tested node ~20 on linux but easy dev docker compiles but what version node is unknown by me. It may be the recommended version to use which is 20.10 but don't know if Brady has updated docker for that version yet but the library is available for linux and should download. https://github.com/marudor/libxmljs2/releases

After testing v20.10.0 and while it compiles for me, serveccda.js doesn't run with 
```
Error: The module '\\?\C:\xampp\htdocs\openemr\ccdaservice\oe-blue-button-generate\node_modules\libxmljs2\build\Release\xmljs.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 108. This version of Node.js requires
NODE_MODULE_VERSION 115. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
```
I'm not sure what this means but I need to figure out why as my readme recommends this version so something has changed.
Still this issue in the end is not a bug and the library should install and why the 404 needs to be investigated Sherwin.

It's not like I have the time to deal with CCDA so thanks Sherwin!:)

**Comment by sjpadgett** [comment]
Oh @juggernautsei 
Try node version 18.20.1
